src/Rector/AbstractDrupalCoreRector.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+declare(strict_types=1);
+
+namespace DrupalRector\Rector;
+
+use Drupal\Component\Utility\DeprecationHelper;
+use DrupalRector\Contract\DrupalCoreRectorInterface;
+use PhpParser\Node;
+use PhpParser\Node\Expr\ArrowFunction;
+use Rector\Core\Rector\AbstractRector;
+
+abstract class AbstractDrupalCoreRector extends AbstractRector implements DrupalCoreRectorInterface
+{
+    public function refactor(Node $node)
+    {
+        $drupalVersion = substr(\Drupal::VERSION, -4) === '-dev' ? str_replace(['.x-dev', '-dev'], '.0', \Drupal::VERSION) : \Drupal::VERSION;
```

Comment:
There is no penalty to run `str_replace` on a non-matching string, so we can simplify this logic and make it easier to read

Reply:
Fixed